modules/op-release-notes-1-6.adoc

```diff
@@ -306,4 +306,32 @@ Error from server (InternalError): Internal error occurred: failed calling webho
 ** For implicit params:
 *** Do not apply the `PipelineSpec` parameters to the `TaskRefs` object.
 *** Disable implicit param behavior for the `Pipeline` objects.
-// https://github.com/tektoncd/pipeline/releases/tag/v0.28.3
\ No newline at end of file
+// https://github.com/tektoncd/pipeline/releases/tag/v0.28.3
+
+[id="release-notes-1-6-3_{context}"]
+== Release notes for {pipelines-title} General Availability 1.6.3
+
+[id="deprecated-features-1-6-3_{context}"]
+=== Deprecated features
+
+* With this update, the {pipelines-title} Operator stops installing pod security policies from components, as the policies are deprecated. The following upgrade paths are affected:
+** On upgrading from {pipelines-shortname} 1.6.1 or 1.6.2 to {pipelines-shortname} 1.6.3, the existing pod security policies from components such as Pipelines and Triggers are deleted.
+** On upgrading from {pipelines-shortname} 1.5.x to 1.6.3, the existing pod security policies are retained. However, cluster administrators can delete them manually.
++
+[NOTE]
+====
+When you upgrade to future releases, the {pipelines-title} Operator will automatically delete all obsolete pod security policies.
+====
+// https://issues.redhat.com/browse/SRVKP-2259
+
+[id="fixed-issues-1-6-3_{context}"]
+=== Fixed issues
+
+* With this update, project administrators can access pipeline metrics in the Developer console.
+// https://issues.redhat.com/browse/SRVKP-2129
+
+* With this update, the reliability and consistency of installing various components of {pipelines-title} are improved.
+// https://issues.redhat.com/browse/SRVKP-2249
+
+* With this update, the speed of lifecycle management of Tekton resources are improved.
+// https://issues.redhat.com/browse/SRVKP-2043
```

Comment:
None of this mentions what was the case before the update. Please rewrite following the guideline prescribed here: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes